frontend/src/components/BattlePlans.svelte

```diff
@@ -1,17 +1,21 @@
 <script>
     import AddPlan from '../components/AddPlan.svelte'
     import HollowButton from '../components/HollowButton.svelte'
+    import JiraImport from '../components/JiraImport.svelte'
 
     export let plans = []
     export let isLeader = false
     export let sendSocketEvent = () => {}
     export let eventTag
+    export let notifications
 
     let showAddPlan = false
     let revisePlanId = ''
     let revisePlanName = ''
     let showCompleted = false
 
+    let allowJiraImport = appConfig.AllowJiraImport
```

Comment:
this value is never changed, should be a **const** not **let**

Reply:
Fixed